bundles/org.openhab.binding.max/src/main/java/org/openhab/binding/max/internal/MaxConsoleCommandExtension.java

```diff
@@ -37,12 +38,13 @@
  * @author Marcel Verpaalen - Initial contribution
  */
 @Component(service = ConsoleCommandExtension.class)
+@NonNullByDefault
 public class MaxConsoleCommandExtension extends AbstractConsoleCommandExtension {
 
     private static final String SUBCMD_BACKUP = "backup";
     private static final String SUBCMD_REBOOT = "reboot";
 
-    private ThingRegistry thingRegistry;
+    private @Nullable ThingRegistry thingRegistry;
```

Comment:
What about using constructor injection for this service implementation and omit `@Nullable` at all but make it `final`?

Reply:
yes, that would be an improvement.
Done